packages/waallet/rpc.ts

```diff
@@ -25,6 +26,11 @@ export type WaalletRequestArguments =
         | WaalletRpcMethod.eth_requestAccounts
     }
 
+export type DefaultArguments = {
+  method: string
+  params?: [any]
+}
```

Comment:
```suggestion
export type RpcRequestArguments = {
  method: string
  params?: any[]
}
```

Reply:
I'm not sure how to adjust here.
If I change `[any]` to `any[]`, `TypeScript` will throw errors at [WaalletProviderL57, L59, L61](https://github.com/pilagod/waallet-extension/blob/753eacb3011cc0b7da22f831afcc9100a5a9790d/packages/waallet/background/provider.ts#L57-L61):
```
Argument of type 'any[] | [{ from?: string; to?: string; gas?: BigNumberish; gasPrice?: BigNumberish; value?: BigNumberish; data?: string; }]' is not assignable to parameter of type '[{ from?: string; to?: string; gas?: BigNumberish; gasPrice?: BigNumberish; value?: BigNumberish; data?: string; }]'.

Type 'any[]' is not assignable to type '[{ from?: string; to?: string; gas?: BigNumberish; gasPrice?: BigNumberish; value?: BigNumberish; data?: string; }]'.

Target requires 1 element(s) but source may have fewer.ts(2345)
```